test/test895.js

```diff
@@ -0,0 +1,115 @@
+if (typeof exports === 'object') {
+	var assert = require('assert');
+	var alasql = require('..');
+}
+
+describe('Test 895 - SERIAL type should not overwrite explicitly provided values', function () {
+	const test = '895';
+
+	before(function () {
+		alasql('create database test' + test);
+		alasql('use test' + test);
+	});
+
+	after(function () {
+		alasql('drop database test' + test);
+	});
+
+	it('A) SERIAL column should auto-increment when not provided', function () {
+		alasql('CREATE TABLE test_serial (id serial, name varchar(50))');
```

Comment:
Please convert multiple alasql calls like this with a multipl line ` enclosed string and commands separated by ;